config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+Sidekiq.configure_server do |config|
+  config.redis = { url: Exercism.config.sidekiq_redis_url }
```

Comment:
I don't know if this `raises` if this value isn't set, but it probably should (or the server is going to hang a bit later, which is a weird error)

Reply:
Good shout.

Reply:
Done in Exercism Config v0.58.